py-polars/polars/expr/expr.py

```diff
@@ -5370,6 +5370,12 @@ def hash(
 
         The hash value is of type `UInt64`.
 
+        .. note::
```

Comment:
This should be in a `Notes` section below the parameters section.

Reply:
Ah ok didn't know that, I just grabbed a random example from the code and followed that.